backends/cuda/ceed-cuda-vector.c

```diff
@@ -189,7 +189,7 @@ static int CeedVectorTakeArray_Cuda(CeedVector vec, CeedMemType mtype,
 //------------------------------------------------------------------------------
 // Set host array to value
 //------------------------------------------------------------------------------
-static int CeedHostSetValue(CeedScalar *h_array, CeedInt length,
+static int CeedHostSetValue_Cuda(CeedScalar *h_array, CeedInt length,
                             CeedScalar val) {
```

Comment:
```suggestion
static int CeedHostSetValue_Cuda(CeedScalar *h_array, CeedInt length,
                                 CeedScalar val) {
```

Reply:
Whoops, fixed in 54e2d42e9d9a0460ed005c56656e5084e1da32c1